openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -405,8 +409,8 @@ def verify_error_response(field_name, data):
                 "Field '{0}' cannot be edited.".format(field_name), data["field_errors"][field_name]["user_message"]
             )
 
-        for field_name in ["username", "date_joined", "is_active"]:
-            response = self.send_patch(client, {field_name: "will_error", "gender": "f"}, expected_status=400)
+        for field_name in ["username", "date_joined", "is_active", "profile_image", "requires_parental_consent"]:
+            response = self.send_patch(client, {field_name: "will_error", "gender": "o"}, expected_status=400)
```

Comment:
Good catch. Did you look through the rest of the places where gender is specified to make sure it differs from our new default gender?